.bin/deploy.sh

```diff
@@ -5,32 +5,21 @@ set -e
 git config --global user.email "travis@browser.html"
 git config --global user.name "travis"
 
-TAG=`git describe --abbrev=0 --tags`
-LATEST_REV=`git rev-parse $TAG`
 REPO_URL=`git config --get remote.origin.url`
 HEAD=`git rev-parse HEAD`
-DEPLOYED_REV=`curl --silent https://raw.githubusercontent.com/browserhtml/browserhtml/gh-pages/HEAD`
+DEPLOY_BRANCH="crate"
 
-if [ "$HEAD" = "$LATEST_REV" ]; then
```

Comment:
> You still need to check that this is running on master branch and do not deploy otherwise or we'll end up deploying on every pull request which is not what we want.

When can this happen? Travis will only call deploy.sh on master when there's a new set of commits, no?